src/MainWindow.vala

```diff
@@ -190,6 +213,35 @@ public class Tasks.MainWindow : Hdy.ApplicationWindow {
         });
     }
 
+    private void action_add_new_list () {
+        Tasks.Application.model.get_registry.begin ((obj, res) => {
+            E.SourceRegistry registry;
+            try {
+                registry = Tasks.Application.model.get_registry.end (res);
+            } catch (Error e) {
+                critical (e.message);
+                return;
+            }
+
+            try {
+                var new_local_source = new E.Source (null, null);
+                E.SourceTaskList source_task_list = (E.SourceTaskList)new_local_source.get_extension (E.SOURCE_EXTENSION_TASK_LIST);
+                source_task_list.backend_name = "local";
```

Comment:
There must be a different name for this because I'm getting duplicate headers when adding a new list. There's `E.SOURCE_EXTENSION_LOCAL_BACKEND` but this seems to be something different and gives us a bunch of errors if we try to use it

Reply:
@danrabbit can you elaborate on what you mean by "duplicate headers"...? I can't see any error/warning outputs using `export G_MESSAGES_DEBUG=all`.

FWIW: I just fixed the two `(io.elementary.tasks:146414): GLib-CRITICAL **: 07:34:25.964: g_utf8_collate: assertion 'str2 != NULL' failed` errors in my latest commit. Apparently we also need to set a `display_name` and a `parent` for the source.

Reply:
Nice catch. I was thinking the sort func probably needs to be able to handle null named lists in case we pull one from elsewhere.

Are you not seeing this?: 
![Screenshot from 2020-10-08 10 35 52@2x](https://user-images.githubusercontent.com/7277719/95494077-2f45bd80-0952-11eb-95b7-c4977309e561.png)

Reply:
Ah this must have to do with invalidating headers because it's fixed on restarting the app